Patches/Alpha Animals/ThingDefs_Races/AlphaAnimals_CE_Patch_Race_CrepuscularBeetle.xml

```diff
@@ -20,13 +20,14 @@
 			<li Class="PatchOperationReplace">
 				<xpath>/Defs/ThingDef[defName="AA_CrepuscularBeetle"]/statBases/ArmorRating_Blunt</xpath>
 				<value>
-					<ArmorRating_Blunt>5</ArmorRating_Blunt>
+					<ArmorRating_Blunt>8</ArmorRating_Blunt>
 				</value>
 			</li>
+
 			<li Class="PatchOperationReplace">
 				<xpath>/Defs/ThingDef[defName="AA_CrepuscularBeetle"]/statBases/ArmorRating_Sharp</xpath>
 				<value>
-					<ArmorRating_Sharp>3</ArmorRating_Sharp>
+					<ArmorRating_Sharp>4</ArmorRating_Sharp>

```

Comment:
3 sharp is respectable for a non-combat insect. I'd leave the sharp as-is, but you could increase the blunt if you'd like.